src/cacheline.c

```diff
@@ -8,6 +8,8 @@
 #include <qthread/cacheline.h>
 #include <qthread/common.h>
 
+#define HAVE_GCC_INLINE_ASSEMBLY
```

Comment:
Is this a fly-by (unrelated) change?

Reply:
I opted to not remove that macro from that specific file so I just defined it at the top instead. The whole logic there needs an overhaul. I'm fine going back and cutting it out there too if that'd be preferable.